pkg/socketservice/socketservice.go

```diff
@@ -144,33 +147,53 @@ func deployLoop() {
 		}
 
 		for _, a := range apps {
-			if err := processDeploySocketForApp(clusterSocket, a); err != nil {
-				logger.Error(errors.Wrapf(err, "failed to run deploy loop for app %s in cluster %s", a.ID, clusterSocket.ClusterID))
-				continue
+			deployed, err := processDeploySocketForApp(clusterSocket, a)
+			if err != nil {
+				_, ok := appDeployLoopErrorBackoff[a.ID]
+				if !ok {
+					appDeployLoopErrorBackoff[a.ID] = &util.ErrorBackoff{MinPeriod: 1 * time.Second, MaxPeriod: 30 * time.Minute}
+				}
+				appDeployLoopErrorBackoff[a.ID].OnError(err, func() {
+					logger.Error(errors.Wrapf(err, "failed to run deploy loop for app %s in cluster %s", a.ID, clusterSocket.ClusterID))
+				})
+			} else if deployed {
+				logger.Infof("Deploy success for app %s in cluster %s", a.ID, clusterSocket.ClusterID)
 			}
 		}
+
+		time.Sleep(5 * time.Second)
```

Comment:
why override the seconds here? and why 5? it's already defined in this step https://github.com/replicatedhq/kots/pull/1723/files#diff-0a0a367f8346de283aa12dd1378bea8f6e6aa93d025aec20683bbf757b912fe7R123

Reply:
i meant to take this out. thanks